js/src/commons/catalog/controllers.coffee

```diff
@@ -74,15 +74,6 @@ module.controller("ProjectSheetCreateCtrl", ($scope, ProjectSheet, Project, Post
                 q_a.projectsheet = projectsheetResult.resource_uri
                 ProjectSheetItem.post(q_a)
             )
-            # add connected user as team member of project with detail "porteur"
```

Comment:
I moved out this part to makerscience/catalog/controllers since it's not using commons based modules